secretsclient/api/api.pb.go

```diff
@@ -0,0 +1,543 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
How is this file generated / updated? Should we place the protos in a more common location where both the server and earth can access?

Reply:
that's a good plan, right now I simply copy and pasted this out of the server repo :(